scope/xgb.py

```diff
@@ -86,7 +86,7 @@ def evaluate(self, X_test, y_test, **kwargs):
         return self.model.eval(dtest, 'dtest', **kwargs)
 
     def load(self, path_model, **kwargs):
-        self.load_model(path_model, **kwargs)
+        self.model.load_model(path_model, **kwargs)
```

Comment:
Any need to try except for both here?

Reply:
That's a good idea, I'll incorporate a try/except loop into this method.